### ISSUE TITLE
Update bounty hunter issue template

### ISSUE THREAD

**Issue by wmcheung** [Project-Catalyst/project-catalyst.github.io#192]
I believe increase increment should be days and not ADA @stephen-rowan @fhilipk @mwojtera thoughts?

**Comment by stephen-rowan** [comment]
I agree that the template parameters should match your bounty process. So if you calculate increments in terms of days then this should be changed. 

Also - A short, one sentence description of why the parameter is needed can also be written under each parameter section (to guide users new to raising bounty issues).

At the end I included a link to a Code of Conduct that does not exist yet on project-catalyst.github.io and a checkbox that has to checked in order to submit the bounty issue. This was to ensure that a user has to state they have read something - it could be bounty procedures or a FAQ instead of a Code of Conduct for example.

The next stage of enhancement would be integration in an Action - e.g. so that the parameter values are sent to a data file or whenever the issue is amended it triggers another stage in the bounty workflow.
